RELEASE_NOTES.md

```diff
@@ -1,68 +1,65 @@
-* Bump `cray-nls` version to 4.0.15
-* Added support to upgrade CSM through IUF
-* Added support for iSCSI based boot content projection for `rootfs` and `PE` images
+> ***`IMPORTANT`*** For systems with NVIDIA CPUs and GPUs must see [Known issues](#known-issues)
 
 ## Improvements
 
-### Automation improvements
-
-* CSM can now be upgraded with IUF.
+### General improvements
+
+* `CASM-5267` Avoid infinite loop while uploading artifacts with cray-nexus-setup image
+* `CASMCMS-9068` Allow customization of ipxe debug options
+* `CASMCMS-9177` Make BOS migration pod more polite
+* `CASMCMS-9188` TESTS: cmsdev: Add explicit check for blank CFS ID field
+* `CASMCMS-9189` Update CFS API spec to reject invalid component creation/update requests
+* `CASMCMS-9197` Bypass needless work in some CFS queries
+* `CASMCMS-9200` Make CFS Options class thread-safe and more performant
+* `CASMCMS-9202` Add ability to create CFS source and specify secret name instead of username/pw
+* `CASMCMS-9207` Update CFS API spec with actual status code for successful source restore
+* `CASMCMS-9208` Decode source_name when restoring source
+* `CASMCMS-9211` Improve CFS config delete performance on scale systems
+* `CASMCMS-9225` Add context managers around BOS requests/sessions; enable paging of BOS components
+* `CASMHMS-6299` PCS/TRS: Mitigate resource leaks / heavy usage
+* `CASMHMS-6360` Make cray-hms-rts version consistent in csm manifest
+* `CASMINST-6968` Replace `sed` calls with `yq` in upgrade_control_plane.sh
+* `CASMINST-7039` Cleanup previous/old SquashFS images during upgrade
+* `CASMMON-463` Update customization.yaml for sma victoria metrics pvc size
+* `CASMPET-7262` TESTS: After installing csm-testing RPM, automatically restart goss-servers
+* `CASMPET-7263` TESTS: goss-servers should delay start until hostname is set
+* `CASMPET-7288` TESTS: Minimize use of system Python packages in virtual environment
+* `CASMPET-7290` TESTS: Add pylint to csm-testing build pipeline
+* `CRAYSAT-1896` Update "sat bootprep" to support CFS v2 or v3
+* `CRAYSAT-1923` Update "sat bootsys" to support CFS v2 or v3
+* `CRAYSAT-1936` Add ability to sort reports by multiple fields
+* `CRAYSAT-1942` Drop internal default values for rootfs_provider{,passthrough} from sat bootprep
+* `MTL-2514` Pin cni / cni-plugins RPM's to the latest version from SLE-Module-Containers
+* `SSI-14310` Update CSM 1.6 configuration page to include USS product changes.
 
 ### Base platform component upgrades
 
 | Platform Component           | Version |
 |------------------------------|---------|
-| `Kubernetes`                 | 1.24.17 |
+| Kubernetes                   | 1.24.17 |
 | `Kyverno`                    | 1.10.7  |
 | `Strimzi Kafka`              | 0.41.0  |
 | `argo-workflow-controller`   | 3.4.5   |
 | `argo-workflows`             | 3.3.6   |
 | `bitnami-etcd` for clusters  | 3.5.9   |
-| `etcd` on `ncn-mxxx`         | 3.5.0   |
+| `bitnami-etcd` on `ncn-mxxx` | 3.5.0   |
```

Comment:
this is bare metal etcd

Reply:
What's it literally called? `bare-metal-etcd`?